aiven/client/cli.py

```diff
@@ -1136,6 +1136,140 @@ def service_acl_list(self):
         self.print_response(service.get("acl", []), json=self.args.json,
                             table_layout=layout)
 
+    @arg.project
+    @arg.service_name
+    def service_connector_available(self):
+        """List available Kafka connectors"""
+        project_name = self.get_project()
+
+        response = self.client.get_available_kafka_connectors(project_name, self.args.name)
+
+        self.print_response(response)
+
+    @arg.project
+    @arg.service_name
+    def service_connector_list(self):
+        """List Kafka connectors"""
+        project_name = self.get_project()
+
+        response = self.client.list_kafka_connectors(project_name, self.args.name)
+
+        self.print_response(response)
+
+    @arg.project
+    @arg.service_name
+    @arg("connector", help="Connector name")
+    def service_connector_status(self):
+        """Get Kafka connector status"""
+        project_name = self.get_project()
+
+        response = self.client.get_kafka_connector_status(project_name, self.args.name, self.args.connector)
+
+        self.print_response(response)
+
+    @arg.project
+    @arg.service_name
+    @arg("connector", help="Connector name")
+    def service_connector_configuration(self):
+        """Get Kafka connector configuration"""
+        project_name = self.get_project()
+
```

Comment:
The generic style of existing code seems to be fairly dense and not having line break here or after the next line would seem to be more consistent.